scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```diff
@@ -59,10 +59,39 @@ private object PairSCollectionFunctions {
     val numBFs = (numEntries / capacity).toInt + 1
 
     val totalBytes = width.toLong * numBFs / 8
-    logger.info(s"Optimal Bloom Filter settings for numEntries = $numEntries, fpProb = $fpProb")
-    logger.info(
-      s"BF width = $width, capacity = $capacity, numBFs = $numBFs, total bytes = $totalBytes"
-    )
+    val totalSizeMb = totalBytes / 1024 / 1024
+
+    /*
+     * For sparse transforms, BloomFilters are stored as SideInputs. For some runners, this side
```

Comment:
@anish749 move this up to this method signature? I think you can also move the comments around the runners limits to it. wdyt?

Reply:
Yes, this sounds great. I'll changed this.